bitcoin/src/crypto/key.rs

```diff
@@ -721,6 +721,10 @@ pub type UntweakedKeypair = Keypair;
 /// let _pk = TweakedPublicKey::from(keypair);
 /// # }
 /// ```
+// #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+// #[cfg_attr(feature = "serde", serde(crate = "actual_serde"))]
+// #[cfg_attr(feature = "serde", serde(transparent))]
```

Comment:
Can you remove the commented out stuff please. I don't understand why line 725 is there, we don't have a "serialize" feature.

Reply:
I was doing some experiments related to them -> but forgot to remove them from PR , My Bad.

Reply:
I mean to say that this PR does not close #1547 because it does not use `i32` (or include some argument as to why the issue is incorrect).  I intentionally did not say which option is correct mainly because I didn't allocate enough review time to fully understanding the issue.

FTR the PR can be useful without closing the issue :)